docs/source/developers/reviewing.rst

```diff
@@ -0,0 +1,226 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+=======================
+Reviewing contributions
+=======================
+
+Principles
+==========
+
+Arrow is a foundational project that will need to evolve over many years
+or even decades, while serving potentially millions of users.  We believe
+being meticulous when reviewing brings greater rewards to the project than
+being lenient and aiming for quick merges.
+
+Fixing potential issues after a Pull Request (PR) merge is more costly since
+it forces the developer to context-switch back to work that was thought
+to be finished.  Moreover, an late-minute API change may propagate to
+other parts of the project and require more attention than if done up front.
+And there are social issues with asking a volunteer to go back to work
+on something that was accepted as finished.
+
+Guidelines
+==========
+
+Meta
+----
+
+**Use your common sense**.  These guidelines are not hard rules.
+Committers are expected to have sufficient expertise on their work
+areas to be able to modulate concerns if necessary.
```

Comment:
"modulate" isn't quite right here, but I'm not sure on the precise point intended so not sure what to suggest in its place, sorry

Reply:
Hmm... "calibrate" perhaps? "adjust"?

Reply:
I think the bit where I'm confused is knowing what is being calibrated/adjusted, and who has concerns about what - "modulate concerns" doesn't quite work.